tests/unittests/Foundation/NSThreadTests.m

```diff
@@ -42,6 +42,12 @@ - (void)countingloop {
 static const double sExpectedLowThreadPriority = 0;
 static const double sExpectedMidThreadPriority = .5;
 
+// A thread never has a set priority. The system can always change it, in fact it will most likely change the priority.
+// Even still, the thread priority should be close enough to the actual requested value so we need to make sure it's sufficiently close.
+// The largest difference between thread priorities in Windows should be .07 units in difference as Windows pigeon holes priorities into
+// certain values.
+static const double sMarginOfError = .07;
+
 TEST(NSThread, Priority) {
     ThreadTestController* testController = [[[ThreadTestController alloc] init] autorelease];
```

Comment:
CI Builds succeed on builds Release: Build 20160803.6  and Debug: Build 20160803.7. No new tests have been added but old tests have been fixed.